lib/news_scraper/trainer/uri_trainer.rb

```diff
@@ -2,12 +2,16 @@ module NewsScraper
   module Trainer
     class UriTrainer
       def initialize(uri)
-        @article_scrape_patterns = Transformers::Article.scrape_patterns
+        @article_scrape_patterns = YAML.load_file(NewsScraper::Constants::SCRAPE_PATTERN_CONFIG_FILE)
         @data_type = NewsScraper::Trainer::PresetsSelector.new(uri)
 
         @uri = uri
         @raw_html = Extractors::Article.new(uri: uri).extract
-        @transformer = Transformers::Article.new(uri: uri, payload: @raw_html)
+        @transformer = Transformers::Article.new(
```

Comment:
I was thinking that we just remove the `@transformer.transform` and the rescue from `def train` and just check for `article_scrape_patterns['domains'].key?` instead. 

That way we can remove the injection of `scrape_pattern` until we absolutely need to.